docs/zkapps/o1js/ecdsa.mdx

```diff
@@ -22,48 +22,50 @@ Please note that zkApp programmability is not yet available on Mina Mainnet, but
 # ECDSA
 
 ECDSA, or Elliptic Curve Digital Signature Algorithm, is a cryptographic algorithm used to sign and verify messages. It is used in many blockchains, including Ethereum, to sign transactions.
-ECDSA works with different elliptic curves but most commonly with the secp256k1 curve. This curve is also used in Bitcoin and Ethereum.
+ECDSA works with different elliptic curves but most commonly with the [secp256k1](/glossary#secp256k1) curve. This curve is also used in Bitcoin and Ethereum.
 
-# Why ECDSA?
+## Why ECDSA?
 
-In order to interact with other blockchains or verify data from the outside world, o1js needs to be able to verify signatures.
-ECDSA is a widely used algorithm that is supported by many libraries and tools. It is also used in Ethereum, so it is important to support it in o1js as well. For example, Ethereum transactions are signed using ECDSA over the secp256k1 curve.
-If a zkApp developer wanted to verify an Ethereum transaction and make a statement about it, they would need to be able to verify the signature of the transaction. This is why ECDSA is important for zkApps.
+To interact with other blockchains and verify data from the outside world, o1js needs to be able to verify signatures.
+ECDSA is a widely used algorithm that is supported by many libraries and tools. For example, Ethereum transactions are signed using ECDSA over the secp256k1 curve.
+As a zkApp developer, when you want to verify an Ethereum transaction and make a statement about it, you must be able to verify the signature of the transaction which is why ECDSA is important for zkApps.
 
-# Basic usage
+## Basic usage
 
-The ECDSA gadget is used to verify ECDSA signatures. It takes as input the message, the signature, and the public key of the signer. It outputs a `Bool` indicating whether the signature is valid or not.
+The ECDSA gadget is used to verify ECDSA signatures. The gadget takes as input the message, the signature, and the public key of the signer. It outputs a `Bool` indicating whether the signature is valid.
 
-Before we can verify a signature, we must initiate the gadget with a curve configuration. The most commonly used curve is secp256k1, which is also used in Ethereum. We can initiate the curve as follows:
+Before you can verify a signature, you must initiate the gadget with a curve configuration. 
```

Comment:
we already stated that sep256k1 is the most commonly used, so let's omit it here